CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - `[jest-environment-jsdom]` [**BREAKING**] Upgrade JSDOM from v11 to v15 ([#8851](https://github.com/facebook/jest/pull/8851))
 - `[jest-util]` [**BREAKING**] Remove deprecated exports ([#8863](https://github.com/facebook/jest/pull/8863))
 - `[jest-validate]` [**BREAKING**] Use ESM exports ([#8874](https://github.com/facebook/jest/pull/8874))
+- `[docs]` Fix minor typo in CHANGELOG and blog post ([#9069](https://github.com/facebook/jest/pull/9069))
```

Comment:
I think we can omit that for a small change like this so the changelog doesn't get too polluted

Reply:
Agreed! Thanks, I'll remove it